sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -90,6 +90,23 @@ static void report_test_status(bool result) {
   test_status_set(result ? kTestStatusPassed : kTestStatusFailed);
 }
 
+// Override the status report function log into on the UART
+void status_report(status_t status) {
+  int32_t arg;
+  const char *message;
+  char mod_id[4];
+  if(status_ok(status)) {
+    LOG_INFO("report status: kOk(%u)", status);
+  }
```

Comment:
do you need this? technically this would only get called on an error no?

Reply:
Normally yes, but the `status_report` function could be called directly. If that ever happened, given that the bitfields of good and bad status are completely different, the output would be very confusing. I'd rather add one line of code and avoid the confusion.

Reply:
Ah right, actually probably would be good to have this still for when you want to use this function to decode any status_t message.